public/js/jibe.js

```diff
@@ -162,6 +162,9 @@ var Jibe = (function (BCSocket, CodeMirror, Replay, showdown, Timestamps, TextFo
    */
   function updatePreview (editor, converter) {
     var preview = $('.rendered-markdown');
+    //May want to update TOC here too
+    toc = new TOC(editor);
+    toc.generateHeaders();
```

Comment:
Rather than creating a new TOC everytime, could you scope TOC outside of this method and then just call toc.generateHeaders?